app/routes/__init__.py

```diff
@@ -1,8 +1,16 @@
-from flask import Blueprint, Flask
+from flask import Flask
 
-bp_api = Blueprint("bp_api", __name__)
+def init_app(app: Flask):
```

Comment:
Using the above approach we could register all blueprints into the bp_api and then register the bp_api to the app, but if yous decided this way would be better, just let me know and i resolve these comments.